include/BSMPT/models/ClassPotentialOrigin.h

```diff
@@ -969,6 +974,21 @@ class Class_Potential_Origin
    */
   Eigen::MatrixXcd LeptonMassMatrix(const std::vector<double> &v) const;
 
+  /**
+   * Ensures the correct rotation matrix convention
+   */
+  virtual void AdjustRotationMatrix() = 0;
+  /**
+   * Returns true if two values are the same given some relative precision
+   */
+  bool almost_the_same(double a, double b, double rel_precision = 0.01);
```

Comment:
Please move the ```almost_the_same``` functions into ```utility.h```/```utility.cpp```.

Reply:
I found that the `almost_the_same` function appeared also in `minima_tracer.cpp`/`minima_tracer.h`, so I moved all the versions of the function (with `double`, `std::complex<double>`, and `std::vector<double>` arguments) to `utility.cpp`/`utility.h`.